.gitignore

```diff
@@ -57,6 +57,10 @@ localhost.crt
 localhost.key
 nscacert.pem
 
+# gov one login keys
+private_key.pem
```

Comment:
Are these in the credentials now?

Reply:
Yes [7313839](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/7313839ff0455bbdda6aa8c474202632476b98ec)